tests/plotting/test_plot_manager.py

```diff
@@ -488,7 +488,7 @@ def raising_callback(event: PlotRefreshEvents, _: PlotRefreshResult):
     assert len(env.refresh_tester.plot_manager.no_key_filenames) == 0
     # The next run without the valid callback should lead to re-loading of all plot
     env.refresh_tester.plot_manager.set_refresh_callback(default_callback)
-    expected_result.loaded = env.dir_1.plot_info_list() + env.dir_2.plot_info_list()
+    expected_result.loaded = env.dir_1.plot_info_list() + env.dir_2.plot_info_list()  # type: ignore[assignment]
```

Comment:
Ditto... why is this here?

Reply:
Adding `chia/plotting/__init__.py` enabled more `mypy` checks in the related code.